lib/src/replaced_element.dart

```diff
@@ -155,7 +155,6 @@ class ImageContentElement extends ReplacedElement {
             return new Image.network(
               src,
               width: snapshot.data.width,
-              height: snapshot.data.height,
```

Comment:
Does this mean that fixed-height images will break? Maybe I missed something higher up in the code, just want to make sure that this doesn't break fixed-height images

Reply:
What do you mean by fixed-height images? As we only supply the width, Flutter will use the natural aspect ratio of the image. But at the same time it will respect the available horizontal space. That's why this fix works (even when using in tables).

Reply:
You mean if you supply the width and height of the image from HTML code? I don't think we supported that ever.

Reply:
@erickok Yes, I mean the height attribute. If we don't support it, then I'm fine with that. I'll get this merged now